src/cli.ts

```
@@ -0,0 +1,156 @@
+typescript
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(ts/no-unused-expressions)